.github/ISSUE_TEMPLATE/data_archival_request.yml

```diff
@@ -8,13 +7,31 @@ body:
   - type: markdown
     attributes:
       value: |
-        Please follow the sections in the template and provide us the information required to retrieve your files! 	
+        Please follow the sections in the template and provide us the required information to retrieve your files! There are more than 10+ hubs serving diverse instructional needs of the campus community. We require you to identify the correct URL to your files in the exact hub used as part of your course.  
   - type: textarea
     attributes:
       label: Email Id 
-      description: Berkeley Email Id which you used to access Datahub!
+      description: Share the Berkeley Email Id which you used to access Datahub!
     validations:
       required: true
+  - type: input
+    attributes:
+      label: Graduation Details
+      description: Did you already graduate?
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Term/Year
+      description: Identify the term of the semester (Fall/Spring/Summer) and the year of your course enrollment
+    validations:
+      required: true     
+  - type: input
+    attributes:
+      label: Login Detail
```

Comment:
can i ask why we are asking this? The identifier provided in the text file should be all that is needed for this, right? It also encodes the name of the hub that is archived, so it *should* say which hub it is.

Reply:
@yuvipanda - My understanding is that these changes will help us have relevant data to respond to the archival request when we encounter scenarios like this - https://github.com/berkeley-dsep-infra/datahub/issues/3487. @felder Please feel free to share more details if required.

Reply:
@yuvipanda The problem is a number of people open file retrieval requests without being directed to do so by the text file.  Thus far most, but not all, of those cases involve folks who have graduated.  There's also often a mix of archived and unarchived files from different hubs/filesystems for these people.  As @balajialg mentioned issue #3487 is a very good example of this issue.  This person had graduated, could no longer login, and had a mix of archived directories as well as directories I had to manually archive and send to them.

Thus I'm asking @balajialg to include language in the template that can steer folks into giving me the correct info.  

@balajialg I'd like to meet with you to discuss.  Like I don't think we should ask if people have graduated specifically.  Instead, I think we should ask something along the lines of whether or not they can still login and direct them to do so if they can.  If they cannot, I'll need information about what hub(s) they used.

Reply:
thank you for pointing out to that issue @felder and @balajialg! Makes sense that is the wrong form to use for this. I'd like to keep this form simple - maybe we can have a different one for people who no longer have access? I'm not sure how this user found this form, but it feels like the problem was that they were completely in the wrong place.

Reply:
@felder Yes, find some slot in my bcal next week to discuss this more. Let's think it through from an user perspective and find an optimal solution :)